components/SSI/DIDxWallet/Wallet/Allowances/index.tsx

```diff
@@ -31,19 +38,49 @@ function Component() {
       setButton("button primary");
       setName("");
```

Comment:
any reason to set it to "" right before assigning it the name? @DevPH-Wistkey

Reply:
No need, because there's no verification like input addr. Already remove setName and setAmount. Thanks!